sdk/go/common/workspace/plugins.go

```diff
@@ -432,7 +432,7 @@ func (source *githubSource) getHTTPResponse(
 		addAuth = " You can set GITHUB_TOKEN to make an authenticated request with a higher rate limit."
 	}
 
-	logging.Errorf("GitHub rate limit exceeded%s%s", tryAgain, addAuth)
+	logging.Errorf("GitHub rate limit exceeded for host%s%s%s", req.URL, tryAgain, addAuth)
```

Comment:
"for host"? maybe just "for".

"iGtHub rate limit exceeded for host https://api.github.com/repos/pulumi/pulumi-mock-latest/releases/latest" vs
"GitHub rate limit exceeded for https://api.github.com/repos/pulumi/pulumi-mock-latest/releases/latest"